tests/test_branch_judge_answers.py

```diff
@@ -10,7 +10,9 @@
     "../taxonomy",
     "main",
 )
-qa_pairs, error_rate = mt_bench_branch.judge_answers("http://localhost:8000/v1")
+qa_pairs, error_rate = mt_bench_branch.judge_answers(
+    api_key="NO_API_KEY", server_url="http://localhost:8000/v1"
```

Comment:
nit: Would you mind ordering these in the same order as in the definition, server_url then api_key.  Same for the other cases in this PR as well.

Reply:
done, updated